server/athenian/api/models/web/for_set_developers.py

```diff
@@ -1,7 +1,8 @@
 from typing import List, Optional
 
 from athenian.api.models.web.base_model_ import AllOf, Model
-from athenian.api.models.web.for_set import CommonPullRequestFilters, RepositoryGroupsMixin
+from athenian.api.models.web.for_set_pull_requests import \
+    CommonPullRequestFilters, RepositoryGroupsMixin
```

Comment:
Let's put `CommonPullRequestFilters` and others to `for_set_common.py` or equivalent.